cosmos-gui/src/app.js

```diff
@@ -49,6 +51,10 @@ var scEndpoint = config.clusters.storage.endpoint;
 var ccPrivKey = config.clusters.computing.private_key;
 var ccUser = config.clusters.computing.user;
 var ccEndpoint = config.clusters.computing.endpoint;
+var httpsOptions = {
+    key: fs.readFileSync(config.private_ley_filekey),
```

Comment:
Yes, there is a typo, I'll fix it.

Regarding the parameters check, I have a tech [debt issue](https://github.com/telefonicaid/fiware-cosmos/issues/30) about it... that I think I will implement in the second release :)